homeassistant/components/mqtt/entity.py

```diff
@@ -477,9 +482,32 @@ def __init__(self, config: ConfigType) -> None:
         self._attributes_sub_state: dict[str, EntitySubscription] = {}
         self._attributes_config = config
 
+    def _update_group_entity_ids(self) -> None:
+        """Set the entity_id property if the entity represents a group of entities.
+
+        Setting entity_id in the extra state attributes will show the discovered entity
+        as a group and will show the member entities in the UI.
+        """
+        if CONF_GROUP not in self._attributes_config:
+            self._default_entity_icon = None
+            return
+        self._attr_icon = self._attr_icon or self._default_group_icon
+        entity_registry = er.async_get(self.hass)
+
+        self._group_entity_ids = []
+        for resource_id in self._attributes_config[CONF_GROUP]:
+            if entity_id := entity_registry.async_get_entity_id(
+                self.entity_id.split(".")[0], DOMAIN, resource_id
+            ):
+                self._group_entity_ids.append(entity_id)
```

Comment:
We need to handle changed entity id of group members

Reply:
Do we want to actively monitor that? IMO it should be enough to do this once when we load the group entity